webview/src/shared/components/sectionContainer/SectionContainer.tsx

```diff
@@ -71,10 +82,10 @@ const InfoIcon = () => (
 )
 
 export const SectionContainer: React.FC<
-  SectionContainerProps<PlotsSection>
+  SectionContainerProps<PlotsSection | SetupSection>
 > = ({
   children,
-  menuItems,
+  menuItems = [],
```

Comment:
[F] We don't need this right now but I thought maybe we could use it for a carousel-like action to move between the screens.

Reply:
`<-` `->` in the icon menu